randovania/data/gui_assets/tracker/prime2-pixel-tournament.json

```diff
@@ -0,0 +1,244 @@
+                "Spider Ball"
+            ],
+            "image_path": "gui_assets/tracker/pixel-icons/mp2/spider_ball.png"
+        },
+        {
+            "row": 4,
+            "column": 8,
+            "resources": [
+                "Violet Translator"
+            ],
+            "image_path": "gui_assets/tracker/pixel-icons/mp2/violet_translator_alt.png"
+        },
+        {
+            "row": 4,
+            "column": 9,
+            "resources": [
+                "Amber Translator"
+            ],
+            "image_path": "gui_assets/tracker/pixel-icons/mp2/amber_translator_alt.png"
+        },
+        {
+            "row": 5,
+            "column": 8,
+            "resources": [
+                "Emerald Translator"
+            ],
+            "image_path": "gui_assets/tracker/pixel-icons/mp2/emerald_translator_alt.png"
+        },
+        {
+            "row": 5,
+            "column": 9,
+            "resources": [
+                "Cobalt Translator"
+            ],
+            "image_path": "gui_assets/tracker/pixel-icons/mp2/cobalt_translator_alt.png"
+        },
+        {
+            "row": 5,
+            "column": 4,
+            "resources": [],
+            "image_path": "gui_assets/tracker/pixel-icons/mp2/dark_agon_key-recolored.png"
+        },
+        {
+            "row": 6,
+            "column": 4,
+            "label": "{capacity}",
+            "style": "font:18pt 'Consolas';color:white",
+            "resources": [
+                "Dark Agon Key 1",
+                "Dark Agon Key 2",
+                "Dark Agon Key 3"
+            ]
+        },
+        {
+            "row": 5,
+            "column": 5,
+            "resources": [],
+            "image_path": "gui_assets/tracker/pixel-icons/mp2/dark_torvus_key-recolored.png"
+        },
+        {
+            "row": 6,
+            "column": 5,
+            "label": "{capacity}",
+            "style": "font:18pt 'Consolas';color:white",
+            "resources": [
+                "Dark Torvus Key 1",
+                "Dark Torvus Key 2",
+                "Dark Torvus Key 3"
+            ]
+        },
+        {
+            "row": 5,
+            "column": 6,
+            "resources": [],
+            "image_path": "gui_assets/tracker/pixel-icons/mp2/ing_hive_key-recolored.png"
+        },
+        {
+            "row": 6,
+            "column": 6,
+            "label": "{capacity}",
+            "style": "font:18pt 'Consolas';color:white",
+            "resources": [
+                "Ing Hive Key 1",
+                "Ing Hive Key 2",
+                "Ing Hive Key 3"
+            ]
+        },
+        {
+            "row": 5,
+            "column": 7,
+            "resources": [],
+            "image_path": "gui_assets/tracker/pixel-icons/mp2/sky_temple_key.png"
+        },
+        {
+            "row": 6,
+            "column": 7,
+            "label": "{capacity}",
+            "style": "font:18pt 'Consolas';color:white",
```

Comment:
might be a dumb thing, but what would happen if `Consolas` is not installed as a font on the system?